inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp

```diff
@@ -180,6 +182,9 @@ ie::ICNNNetwork::Ptr FrontEnd::convertNetwork(ie::ICNNNetwork& network) {
     manager.register_pass<ngraph::pass::ConvertOpSet3ToOpSet2>();
     manager.register_pass<ngraph::pass::ConvertOpSet2ToOpSet1>();
     manager.register_pass<ngraph::pass::ConvertOpSet1ToLegacy>();
+
+    manager.get_pass_config()->disable<ngraph::pass::HSwishDecomposition>();
```

Comment:
Can we extend `transformationsPredicate` instead ? (for example, as for the decomposition of `Gelu`)

Reply:
I specifically chose this method because I'm afraid that other HSwish transformations may appear. But we need only the disabling of decomposition.